requirements-dev.txt

```diff
@@ -1,6 +1,7 @@
 ansible==9.5.1
-ansible-compat==24.5.1
+ansible-compat==24.6.1
 ansible-lint==24.5.0
-molecule==24.2.1
+molecule==24.7.0
 molecule-docker==2.1.0
 yamllint==1.35.1
+requests==2.31.0
```

Comment:
Is this still needed after updating Ansible/molecule and `community.docker`?

Reply:
It is on my laptop 😁

Reply:
Without it I get:

```
"Error connecting: Error while fetching server API version: Not supported URL scheme http+docker"
```